lib/rules/font-family-name-quotes/index.js

```diff
@@ -102,12 +132,22 @@ const rule = (primary) => {
 			const hasQuotes = rawFamily.startsWith("'") || rawFamily.startsWith('"');
 
 			// Clean the family of its quotes
-			const family = rawFamily.replace(/^['"]|['"]$/g, '');
+			const family = rawFamily.replace(/^['"]|['"]$/g, ''); // NOTE: wth??
 
 			// Disallow quotes around (case-insensitive) keywords
 			// and system font keywords in all cases
 			if (keywordSets.fontFamilyKeywords.has(family.toLowerCase()) || isSystemFontKeyword(family)) {
 				if (hasQuotes) {
+					if (context.fix) {
+						const openIndex = fontFamilyNode.sourceIndex;
+						const closeIndex = openIndex + rawFamily.length - 1;
+
+						return {
+							type: 'remove',
+							indexes: [openIndex, closeIndex],
+						};
```

Comment:
[suggest] It seems a bit hard to read to me that `checkFamilyName()` returns an object or `undefined`. 🤔 

For example, can you call a helper function like `removeQuotes()` as below (`addQuotes()` as well)?

```suggestion
						removeQuotes(decl, fontFamilyNode);
						return;
```

Reply:
I'm currently working on this. the declaration value can have multiple font-families. If I add quotes or remove quotes from the `decl.value` the indexes in the reset of the following `fontFamilyNode`s will be offset and be incorrect. so I'm currently still working on this in my free time.